### DISCUSSION TITLE
HERO11 Start on USB Power

### DISCUSSION THREAD

**Discussion by oreixa** [gopro/labs#396] [Q&A] [answered]
HI
I'm using a brand new Hero 11 Black with latest labs firmware (2.10.70) and couldn't configure it to wake and start recording a video when there is power on USB port. Tried several ways like the qr https://gopro.github.io/labs/control/usb/ or !MWAKE or the new >u but don't work. What's the best way to do this? What am I doing wrong?
Many thanks

**Comment by kanishk619** [comment]
!MWAKE=2!MTUSB=1!MBOOT="!LRecOnUsb"!SAVERecOnUsb=>u0!S+"StartRecord"~"zzZzzzZzz"

This should be a good starting point for you. Make sure to scan !RESET QR code with your camera first

**Reply by oreixa** [reply]
Thanks, will try!

**Reply by oreixa** [reply]
I understand that activate WAKE and TUSB and then make a script that loads on boot, that script uses the conditional u to start recording. Isn't it? Seems that dont work, when I connect usv cable only the led of battery charging starts.

**Reply by oreixa** [reply]
Tried this, to use new conditional u format:
!MWAKE=2!MTUSB=1!MBOOT="!LRecOnUsb"!SAVERecOnUsb=>u!S~!15E

But nothing happens :(

**Reply by oreixa** [reply, marked as answer]
Found it!
Added a !R and now works:

!MWAKE=2!MTUSB=1!MBOOT="!LRecOnUsb"!SAVERecOnUsb=>u!S~!15E!R
